portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
```

Comment:
You've actually removed the greeting-container div that this script uses, so it no longer does anything. You may want to consider removing the inclusion of this script from your HTML file.

Reply:
okay, just done that.